recipes/mxnet/meta.yaml

```diff
@@ -0,0 +1,145 @@
+    folder: 3rdparty/dmlc-core
+  - url: https://github.com/dmlc/ps-lite/archive/34fd45c.tar.gz
+    sha256: ec5d5baab8bbf0c3983ad5f18d7f963f15ae7cd4d154ec204b03c1dceccf209b
+    folder: 3rdparty/ps-lite
+  - url: https://github.com/dmlc/dlpack/archive/3efc489.tar.gz
+    sha256: b59586ce69bcf3efdbf3cf4803fadfeaae4948044e2b8d89cf912194cf28f233
+    folder: 3rdparty/dlpack
+  - url: https://github.com/google/googletest/archive/eb9225c.tar.gz
+    sha256: a4cb4b0c3ebb191b798594aca674ad47eee255dcb4c26885cf7f49777703484f
+    folder: 3rdparty/googletest
+  - url: https://github.com/apache/incubator-tvm/archive/efdac94.tar.gz
+    sha256: 656c38d66baeab885b0212602148bb55ad3cf7d22f52ded8ece53f88e103a2f5
+    folder: 3rdparty/tvm
+  - url: https://github.com/onnx/onnx-tensorrt/archive/2eb74d9.tar.gz
+    sha256: df99819727445c247fb5c21c2fd825ded3269376867457ae84fa6d6f1c0ae331
+    folder: 3rdparty/onnx-tensorrt
+  - url: https://github.com/NVlabs/cub/archive/0158fa1.tar.gz
+    sha256: 43424c4c17a997d1d730c89ec14688671245de7941e02b388d7d3df6ea53777a
+    folder: 3rdparty/nvidia_cub
+  - url: https://github.com/nih-at/libzip/releases/download/v1.8.0/libzip-1.8.0.tar.gz
+    sha256: 30ee55868c0a698d3c600492f2bea4eb62c53849bcf696d21af5eb65f3f3839e
+    folder: 3rdparty/libzip
+  - url: https://github.com/kpu/intgemm/archive/8f28282.tar.gz
+    sha256: bc8bd8015613a13747eb769876385ec53e8c1ea7ae3f8414521dc53b8fcdfc65
+    folder: 3rdparty/intgemm
+  - url: https://github.com/oneapi-src/oneDNN/archive/f40443c.tar.gz
+    sha256: da1dd1bb68ce3318db11e57971f975653e84adcb7d65cd879e558f2245e6de9a
+    folder: 3rdparty/onednn
+
+build:
+  number: {{ build_number }}
+  skip: True  # [win or cuda_compiler_version == "10.2"]
+  string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ proc_type }}  # [cuda_compiler_version == "None"]
+  string: py{{ CONDA_PY }}{{ cuda_compiler_version|replace('.', '') }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ proc_type }}  # [cuda_compiler_version != "None"]
+  run_exports:
+    # ensure matching proc-type
+    - mxnet =*=*_{{ proc_type }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - openblas
+    - make
+    - cmake
+    - ninja
+    - ccache
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - python
+    - cython
+    - pip
+    - setuptools
+    - libjpeg-turbo
+    - pkgconfig
+    - pkg-config
+
+  host:
+    # need to better expose these deps
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+    - openblas
+    - mkl  # [(target_platform != "osx-arm64") and (cuda_compiler_version == "None")]
+    - mkl-include  # [(target_platform != "osx-arm64") and (cuda_compiler_version == "None")]
+    - opencv
+    - onednn
+    - cudnn  # [cuda_compiler_version != "None"]
+    - nccl  # [cuda_compiler_version != "None"]
+    - numpy >=1.17
```

Comment:
Numpy is quite forward compatible, but it isn't really backward compatible. Doing this kind of pinning uses 
```
2022-01-22T08:20:42.7430135Z   + numpy                            1.21.5  py37hf2998dd_0        conda-forge/linux-64       6 MB
```
which makes it only compatible with numpy 1.21.5

Please follow
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
So, should I just `numpy ==1.17.*` or should I just drop the pinning here?

Reply:
I actually don't if numpy needs to be in this section... I will need to check that too!

Reply:
just drop the pin. it will get a newer number from the config.

Reply:
OK, will remove. I now think numpy should only be in run... testing this takes forever, that's why I am hesitating to keep going back and forth, but I will try tweaking things around and tidying up locally first.

Reply:
Thanks a lot! 🥇